docs/changelog.md

```diff
@@ -4,6 +4,19 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.3.1 - Jun 23, 2022
+
+### Major Features and Improvements
```

Comment:
```suggestion
```

In v2.2.1 we just skipped this part, so I think we should be consistent with it.